jscomp/bin/bsb.ml

```diff
@@ -9698,14 +9698,14 @@ let output_ninja
     let () =
       output_string oc ninja_required_version ;
       output_string oc "bs_package_flags = ";
-      output_string oc ("-bs-package-name "  ^ package_name);
+      output_string oc ("-bs-package-name "  ^ (Filename.quote package_name));
```

Comment:
I am a little confused since package_name does not allow whitespace while  quoting is still necessary?
btw, would you mind make this PR complete to solve the whietspace problem to tell the whole story? Thanks !